client/app/list-students/list-students.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+angular.module('app')
+	.service('listStudents', function($http) {
```

Comment:
Let's just make a `Student` service. The student service can have a `list` function to get the list of students.